rootex/framework/scene.h

```diff
@@ -36,6 +36,8 @@ class Scene
 	Scene* m_ParentScene = nullptr;
 	Vector<Ptr<Scene>> m_ChildrenScenes;
 
+	static Vector<Scene*> m_Scenes;
```

Comment:
`s_Scenes`
Also, this should be moved to the top of the private section, just by conventions.